plugins/inputs/opcua/sample.conf

```
@@ -48,14 +48,24 @@
   ## namespace         - OPC UA namespace of the node (integer value 0 thru 3)
   ## identifier_type   - OPC UA ID type (s=string, i=numeric, g=guid, b=opaque)
   ## identifier        - OPC UA ID (tag as shown in opcua browser)
-  ## tags              - extra tags to be added to the output metric (optional)
+  ## tags              - extra tags to be added to the output metric (optional); deprecated in 1.25.0; use default_tags
+  ## default_tags      - extra tags to be added to the output metric (optional)
   ## Example:
-  ## {name="ProductUri", namespace="0", identifier_type="i", identifier="2262", tags=[["tag1","value1"],["tag2","value2"]]}
+  ## {name="ProductUri", namespace="0", identifier_type="i", identifier="2262",
+  ##  tags=[["tag1","value1"],["tag2","value2]], default_tags={"tag1"="value1","tag2"="value2}}
```

I would leave out the deprecated `tags` in the sample, it will confuse people.
```suggestion
  ## {name="ProductUri", namespace="0", identifier_type="i", identifier="2262", default_tags={"tag1"="value1","tag2"="value2}}
```